src/styles/cards.scss

```diff
@@ -0,0 +1,137 @@
+.page {
+  display: flex;
+  margin-right: 0;
+  margin-top: 51px;
+  margin-bottom: 51px;
```

Comment:
block in BEM can't have outer margins, use mix class

https://en.bem.info/methodology/css/#mixes